spec/system/change_owner_spec.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+describe "Change Owner Modal", type: :system do
+  fab!(:post) { Fabricate(:post, raw: "This is some post to change owner for") }
+  fab!(:other_user) { Fabricate(:user) }
+  fab!(:admin) { Fabricate(:admin) }
+  let(:user) { post.user }
+  let(:topic) { post.topic }
+  let(:topic_page) { PageObjects::Pages::Topic.new }
+  let(:change_owner_modal) { PageObjects::Modals::ChangeOwner.new }
+
+  before { sign_in(admin) }
+
+  def visit_topic_and_open_change_owner_modal(post)
+    topic_page.visit_topic(topic)
+    topic_page.expand_post_actions(post)
+    topic_page.expand_post_admin_actions(post)
+    topic_page.click_post_admin_action_button(post, :change_owner)
+  end
```

Comment:
Small suggestion but I feel like this can just go into the `PageObjects::Pages::Topic` class instead.